src/l3.c

```diff
@@ -35,12 +44,23 @@ struct l3_entry
     pid_t       tid;
 #ifdef L3_LOC_ENABLED
     loc_t       loc;
+#else
+    uint32_t    loc;
```

Comment:
To simplify unpacking this field from the Python script, always manifest this field by its name.

This way, the `l3_dump.py` does not have "know" if the dump file was created with LOC ON or OFF. 

If L3_LOC_ENABLED is OFF, the field will guaranteedly be 0, so the dumper will skip reporting that value.